sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/QueueDescriptionExtensions.cs

```diff
@@ -94,50 +101,56 @@ private static QueueDescription ParseFromEntryElement(XElement xEntry)
                     case "RequiresSession":
                         qd.RequiresSession = Boolean.Parse(element.Value);
                         break;
+                    case "DefaultMessageTimeToLive":
+                        qd.DefaultMessageTimeToLive = XmlConvert.ToTimeSpan(element.Value);
+                        break;
                     case "DeadLetteringOnMessageExpiration":
                         qd.EnableDeadLetteringOnMessageExpiration = Boolean.Parse(element.Value);
                         break;
                     case "DuplicateDetectionHistoryTimeWindow":
                         qd.duplicateDetectionHistoryTimeWindow = XmlConvert.ToTimeSpan(element.Value);
                         break;
-                    case "LockDuration":
-                        qd.LockDuration = XmlConvert.ToTimeSpan(element.Value);
-                        break;
-                    case "DefaultMessageTimeToLive":
-                        qd.DefaultMessageTimeToLive = XmlConvert.ToTimeSpan(element.Value);
-                        break;
                     case "MaxDeliveryCount":
                         qd.MaxDeliveryCount = Int32.Parse(element.Value);
                         break;
                     case "EnableBatchedOperations":
                         qd.EnableBatchedOperations = Boolean.Parse(element.Value);
                         break;
-                    case "Status":
-                        qd.Status = (EntityStatus)Enum.Parse(typeof(EntityStatus), element.Value);
-                        break;
-                    case "AutoDeleteOnIdle":
-                        qd.AutoDeleteOnIdle = XmlConvert.ToTimeSpan(element.Value);
+                    case "AuthorizationRules":
+                        qd.AuthorizationRules = AuthorizationRules.ParseFromXElement(element);
                         break;
-                    case "EnablePartitioning":
-                        qd.EnablePartitioning = bool.Parse(element.Value);
+                    case "IsAnonymousAccessible":
```

Comment:
Need to double check the order here between IsAnonymousAccessible and AuthRules. In the example linked above, they are swapped.